css/localgov_forms_review_element.css

```diff
@@ -0,0 +1,41 @@
+.webform-localgov-review-element {
+  display: flex;
+  margin-bottom: var(--spacing);
```

Comment:
These variables are only available when localgov_base is used as the base theme. If this component is used anywhere independently, then these variables will all return empty.

Let's update this to use hardcoded values of `1rem` instead of `var(--spacing)` and `1px solid #cecfd0` for border.

We can create a follow-up issue in localgov_base to rewrite this CSS using variables that subthemes can override.